README.md

```diff
@@ -22,8 +24,10 @@ def deps do
   ]
 end
 ```
-### 2. Go to <a href="https://app.configcat.com/sdkkey" target="_blank">Connect your application</a> tab to get your *SDK Key*:
-![SDK-KEY](./media/readme01.png  "SDK-KEY")
+
+### 2. Go to [Connect your application](https://app.configcat.com/sdkkey) tab to get your _SDK Key_:
+
+![SDK-KEY](assets/readme01.png "SDK-KEY")
```

Comment:
They call this folder `media/` on all SDKs. So I named the same intentionally :)

Reply:
See discussion above.